src/api/index.ts

```diff
@@ -1,22 +1,22 @@
-import base_WebpackApi, { WebpackApi } from "./Webpack/index.js";
+import IBaseWebpackApi, { WebpackApi } from "./Webpack/index.js";
 
-export { base_WebpackApi, WebpackApi };
+export { IBaseWebpackApi, WebpackApi };
 
-export function pathToFunction(path: string) {
+export function getFunctionByPath(path: string) {
     const final: () => any = new Function("return () => " + path)();
     Object.defineProperty(final, "path", {
         value: path,
     });
     return final;
 }
 
-export function pathToFunction2(object: string, property: string) {
-    const final: () => any = new Function(`return () => ${object}.${property}`)();
+export function getFunctionByObjectAndProperty(objectName: string, property: string) {
```

Comment:
it doesn't get a function by object and property, it's more like a fake function resolver that resolves to object and property that *should* contain the real function after compilation

Reply:
yean, didn't know how to call it, was just sure that `pathToFunction2` wasn't the right name hahaha